samples/lightweight-component/lightweight_component.ipynb

```diff
@@ -0,0 +1,359 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lightweight python components example\n",
+    "\n",
+    "This pipeline defines two simple math operation: `add` adds two input arguments and returns the sum. `my_divmod` divides two input arguments and returns the quotient and remainder. You can visit [Build Lightweight Python Components](https://www.kubeflow.org/docs/pipelines/sdk/lightweight-python-components/) for this pipeline's detail."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can successfully run this notebook, you can see the result from `Tekton` UI. You can follow [tekton dashboard](https://github.com/tektoncd/dashboard#accessing-the-dashboard) to install the `Tekton` UI. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here are some screenshots to show this pipeline example and result."
```

Comment:
Since these images only available after the pipeline execution, can we move these images to the end of the notebook?

Reply:
ok